src/app/actions/Actions.js

```diff
@@ -1,18 +1,11 @@
-// ACTIONS
-import alt from '../alt.js';
-
-class Actions {
-  searchByKeyword(data) {
-    return data;
-  }
-
-  searchByTitle(data) {
-    return data;
-  }
-
-  searchByAuthor(data) {
-    return data;
-  }
-}
-
-export default alt.createActions(Actions);
+// Actions
+export const Actions = {
```

Comment:
_I think_ it would be okay to have both the constants and the actual action functions in the same class or file.